docs/team/hynridge.md

```diff
@@ -57,7 +57,7 @@ Given below are my contributions to the project.
     [3](https://github.com/HynRidge/ped/issues/3),
     [4](https://github.com/HynRidge/ped/issues/2))
     
-## \[Optional\] Contributions to the User Guide  (Extract)
+## Contributions to the User Guide  (Extract)
```

Comment:
We should remove "Optional" here?

Reply:
I think..if not it looks weird @Ma-Yueran

Reply:
ok, I will change too